test/scripts/get-missing-cherry-picks.test.ts

```diff
@@ -1,150 +1,222 @@
 import assert from 'assert';
-import nock from 'nock';
+import {Octokit} from '@octokit/rest';
+import {GetResponseTypeFromEndpointMethod} from '@octokit/types';
+import {spy, when, deepEqual, resetCalls, reset} from 'ts-mockito';
 import {getMissingCommits} from '../../scripts/get-missing-cherry-picks-utils';
 
-const url = 'https://api.github.com';
 describe('cases where cherry-picks are missing', () => {
-  before(() => {
-    nock.disableNetConnect();
-  });
+  const octokit = new Octokit();
+  const spiedRepos = spy(octokit.rest.repos);
+  type CompareResponse = GetResponseTypeFromEndpointMethod<
+    typeof octokit.rest.repos.compareCommitsWithBasehead
+  >;
```

Comment:
Supernit: move after imports, before describe

Reply:
can't, this uses an instance of octokit.